dough-reflection/src/main/java/io/github/bakedlibs/dough/reflection/ReflectionUtils.java

```diff
@@ -322,9 +314,12 @@ public static Field getField(@Nonnull Class<?> c, @Nonnull String field) throws
      * 
      * @return The Class in that Package
      */
-    @Nonnull
-    public static Class<?> getOBCClass(@Nonnull String name) throws ClassNotFoundException {
-        return Class.forName("org.bukkit.craftbukkit." + CURRENT_VERSION + '.' + name);
+    public static @Nonnull Class<?> getOBCClass(@Nonnull String name) throws ClassNotFoundException {
+        try {
+            return Class.forName("org.bukkit.craftbukkit." + getVersionSpecificPackage(false) + '.' + name);
```

Comment:
As with earlier, if we cached the version string we wouldn't need to catch the `UnknownServerVersionException here`